services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -32,13 +32,57 @@ export const dynamicUiWorkflowDefinition = {
       collection_flow: {
         tags: [StateTag.COLLECTION_FLOW],
         on: {
-          COLLECTION_FLOW_FINISHED: 'run_kyb_enrichment',
+          COLLECTION_FLOW_FINISHED: [
+            {
+              target: 'run_cn_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.country == `CN`',
+                },
+              },
+            },
+            {
+              target: 'run_hk_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.country == `HK`',
+                },
+              },
+            },
+            {
+              target: 'failed',
+            },
+          ],
+        },
+      },
+      run_cn_kyb: {
+        tags: [StateTag.COLLECTION_FLOW],
```

Comment:
it should be something else, not COLLECTION_FLOW - maybe ask Nitzan what we should show on that stages